api/app/Models/User.php

```diff
@@ -348,6 +348,40 @@ public function filterByClassifications(Builder $query, array $classifications):
             });
 
             $this->orFilterByClassificationToSalary($query, $classifications);
+           $this->orfilterByClassificationToGenericJobTitles($query, $classifications);
+
+
+        });
+
+        return $query;
+    }
+     public function orfilterByClassificationToGenericJobTitles(Builder $query, array $classifications): Builder
```

Comment:
Follow casing convention for method names
```suggestion
           $this->orFilterByClassificationToGenericJobTitles($query, $classifications);


        });

        return $query;
    }
     public function orFilterByClassificationToGenericJobTitles(Builder $query, array $classifications): Builder
```